src/app/visrec/visrec.service.js

```diff
@@ -151,10 +151,8 @@ angular.module('voyager')
       var clusters = cp.cluster(encodings)
         .map(function(cluster) {
           return cluster.map(function(spec) {
-            // auto sort
             spec = _.cloneDeep(spec);
```

Comment:
Do we need the copy? 

Also, one blank line is enough. LGTM otherwise. Feel free to merge.